ssn/rdf/sosa.ttl

```diff
@@ -168,6 +168,38 @@ A Sample is intended to sample some FatureOfInterest in an application domain, s
   skos:example "A statistical sample is often designed to be characteristic of an entire population, so that observations can be made regarding the sample that provide a good estimate of the properties of the population."@en ;
   skos:note "A transient sample, such as a ships-track or flight-line, might be identified and described, but is unlikely to be revisited exactly."@en ;
 .
+sosa:Sampler
+  rdf:type rdfs:Class ;
+  rdf:type owl:Class ;
+  rdfs:label "Sampling device"@en ;
```

Comment:
Sampling device --> maybe Sampling entity? I would try to avoid 'device' based on today's discussion.